### PR TITLE
PEP-8 pulp.server.managers.repo.

### PR DESCRIPTION
**Pull request by rbarlow**

### CODE REVIEW

File: server/pulp/server/managers/repo/group/cud.py [hunk 1]
```diff
@@ -1,16 +1,4 @@
...
-# This software is licensed to you under the GNU General Public License as
-# published by the Free Software Foundation; either version 2 of the License
-# (GPLv2) or (at your option) any later version.
-# There is NO WARRANTY for this software, express or implied, including the
-# implied warranties of MERCHANTABILITY, NON-INFRINGEMENT, or FITNESS FOR A
-# PARTICULAR PURPOSE.
-# You should have received a copy of GPLv2 along with this software; if not,
-# see http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt
-
+from gettest import gettext as _
```
Comment (asmacdo) [inline server/pulp/server/managers/repo/group/cud.py]: s/gettest/gettext/

**Comment by asmacdo** [comment]
There is another test that isn't passing, but it might be due to the gettest typo.

**Comment by rbarlow** [comment]
@asmacdo It turned out that I needed to rebase on master due to the new gofer stuff. All tests pass now. Can you recheck? Thanks!

**Comment by asmacdo** [comment]
LGTM

**Pull request merged**